libpkg/pkg_jobs.c

```diff
@@ -1997,6 +1997,14 @@ pkg_jobs_solve(struct pkg_jobs *j)
 
 	pkgdb_end_solver(j->db);
 
+	return ret;
```

Comment:
BSD style puts parens around the return value, though that's inconsistently followed here.

Reply:
Thanks, it seems I haven't gotten that bit of style internalized yet :)